js/rank_states_utils.js

```diff
@@ -179,18 +191,29 @@ function rankEm(barData) {
     }
     rankSvg.updateStyles();
   };
-  
+  var width = rankSvg.desktopWidth;
+  var height = rankSvg.desktopHeight;
+  if (rankSvg.mobile){
```

Comment:
I think you can grab `waterUseViz.mode` which will be either `desktop` or `mobile`. That's a global var, so it should be available here, too.